.github/workflows/docfx.yml

```diff
@@ -0,0 +1,20 @@
+name: docfx
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
Eventually we will have a push trigger here which will publish the generated docs to the document site.